mods/ts/rules/defaults.yaml

```diff
@@ -417,6 +417,8 @@
 		Weapons: SmallDebris
 		Pieces: 3, 7
 		Range: 2c0, 5c0
+	PoisonedByTiberium:
+		Resources: Veins
```

Comment:
This will damage vehicles with the "Tiberium" weapon, which probably doesn't make sense.